tao/tao_bridge/ral/CMakeLists.txt

```diff
@@ -147,10 +147,15 @@ if(${TAO_ENABLE_PATINE})
   target_link_libraries(ral PRIVATE patine_client)
 endif()
 
+if(${ENABLE_BLADE_GEMM})
+  include_directories(${PLATFORM_ALIBABA_DIR}/third_party/blade_gemm/include)
```

Comment:
nit: Is it better to have a prefix in include path like `blade_gem`, so that we include like 
```c++
#include "blade_gemm/blade_gemm.h"
```
instead of 
```c++
#include "blade_gemm"
```
No strong opinion ...

Reply:
Done.